hypervisor/include/arch/x86/vtd.h

```diff
@@ -543,7 +543,8 @@ struct iommu_domain;
  *
  * Remove the device from the fallback iommu domain (if present), and add it to the specific domain.
  *
```

Comment:
Need to update the description of the API. Also, as mentioned in email, please add a note about the device managed by the dmar unit, which is ignored.

Reply:
Please check now.